src/bacnet/basic/bbmd6/h_bbmd6.c

```diff
@@ -1059,6 +1059,7 @@ int bvlc6_handler(
 int bvlc6_register_with_bbmd(
     const BACNET_IP6_ADDRESS *bbmd_addr, uint16_t ttl_seconds)
 {
+    int len;
     uint8_t mtu[BIP6_MPDU_MAX] = { 0 };
     uint16_t mtu_len = 0;
     uint32_t vmac_src = 0;
```

Comment:
I'd have had the initialised right away and in the right order if possible i.e uint32_t vmac_src = Device_Object_Instance_Number();;